src/index.js

```diff
@@ -62,11 +62,11 @@ export default class ReactImgix extends Component {
       height: node.scrollHeight,
       mounted: true
     })
-  }
+  };
```

Comment:
Hmm, that doesn't seem to be the case in 6.4.5, which is the version react-imgix is currently on. Even on a clean checkout, `npm run test` gives:

```
SyntaxError: /Users/paulstraw/dev/imgix/react-imgix/src/index.js: A semicolon is required after a class property (65:3)
  63 |       mounted: true
  64 |     })
> 65 |   }
```